index.js

```diff
@@ -47,12 +81,32 @@ getUsersCreditDetails(usersArray);
 // ***************************************************************************
 // Iteration 4 - practice `.filter()` method and how to return two elements
 // ***************************************************************************
-
+console.log("===== Iteration 4 ====== ");
 const genderView = users => {
   // Your code goes here ...
+  let femaleUsers = users.filter((user) => {
+    if (user.gender === 'female') {
+      return user;
+    }
+  }).map(user => {
+    return (user.firstName + " " + user.lastName);
+  });
+
+  let maleUsers = users.filter((user) => {
+    if (user.gender === 'male') {
+      return user;
+    }
+  }).map(user => {
+    return (user.firstName + " " + user.lastName);
+  });
+
+  let gendersObj = {};
+  gendersObj.femaleUsers = femaleUsers;
+  gendersObj.maleUsers = maleUsers;
+  return gendersObj;
```

Comment:
This is totally correct: There are two faster ways of writing this

```js
let gendersObj = {
    femaleUsers,
    maleUsers
}
```
This way you create the object directly and then return it.

Or, for even faster typing and less prone to errors:

```js
return {
    femaleUsers,
    maleUsers
}
```